frigate/api/defs/request/export_recordinds_body.py

```diff
@@ -0,0 +1,15 @@
+from typing import Union
+
+from pydantic import BaseModel, Field
+from pydantic.json_schema import SkipJsonSchema
+
+from frigate.record.export import PlaybackFactorEnum
+
+
+class ExportRecordingsBody(BaseModel):
+    playback: PlaybackFactorEnum = Field(
+        default=PlaybackFactorEnum.realtime, title="Playback factor"
+    )
+    source: str = "recordings"
+    name: str = Field(title="Friendly name", min_length=1, max_length=256)
```

Comment:
Please be more careful when defining these, `name` can be empty and is optional

Reply:
Yeah, looking back your're right. Will change